src/asInput/index.jsx

```diff
@@ -181,6 +181,11 @@ const asInput = (WrappedComponent, inputType = undefined, labelFirst = true) =>
       );
     }
 
+    genInputGroupAddonKey(prefix, index) {
```

Comment:
I also tend to like having my functions take objects like

```javascript
generateInputGroupAddonKey({ prefix, index })
```

so that when you call this method elsewhere like

```javascript
generateInputGroupAddonKey({ prefix: 'jae', index: 'baebae' })
```

it's clearer what the arguments mean

You can also do stuff like

```javascript
generateInputGroupAddonKey({ index: 'baebae', prefix: 'jae' })
```

and not have to worry about argument order.

Here's [a good article](https://medium.freecodecamp.org/elegant-patterns-in-modern-javascript-roro-be01e7669cbd) on this pattern.